### ISSUE TITLE
Chameleon not getting into Bootloader in Windows 10

### ISSUE THREAD

**Issue by nov3mb3r** [emsec/ChameleonMini#270]
After following all the steps, installing the Chameleon driver, teraterm and dfu programmer, i have a problem entering the bootloader. I press the RButton, upon connecting my USB, but chameleon lights never change. Is this supposed to be like that or could my chameleon be faulty?? I used 2 different computers and installed drivers, but i can't get into bootloader.
When I reboot my computers, The Chameleon driver returns a code 10 error
![image](https://user-images.githubusercontent.com/10418973/90069827-6137fc00-dce2-11ea-8ab7-fb613f2159e0.png)

**Comment by madalinamarin** [comment]
I had the same problem, I solved it using macOS.

**Comment by david-oswald** [comment]
Dear @nov3mb3r, can you check in the dev mngr that the atmel drivers are installed and that the chameleon is really in the bootloader (i.e. shows up as Atmel DFU or so)?

**Comment by nov3mb3r** [comment]
This is my Device Manager:
![image](https://user-images.githubusercontent.com/10418973/90241671-1077ee80-de1b-11ea-940b-74aa67eee6c7.png)
I would expect that the bootloader is the AT32UC3A. Is that correct? In my understanding, drivers are installed but somehow i can't get to boot

**Comment by timokasper** [comment]
are you using an original ChameleonMini RevG? Then this should be an ATXMEGA128A4U. For other devices please ask the manufacturer.

**Comment by nov3mb3r** [comment]
@timokasper As I said in the first issue, I use an original ChameleonMini RevG. I bought it from your workshop in 36c3

**Comment by timokasper** [comment]
just to be sure: the chameleon is switched OFF, then you hold the Rbutton, then you switch ON again ; as a result no LED lights up (indicating the bootloader is active) and then you plug in the USB, correct?

**Comment by nov3mb3r** [comment]
Correct!

**Comment by timokasper** [comment]
![grafik](https://user-images.githubusercontent.com/5642831/90243842-e5e46080-de2f-11ea-9238-da1187855d41.png)

I did the same it looks like this. Windows 10 64 bit.

so for some unknown reason you have a different driver. I had installed a while ago the WinAVR package is that maybe the cause?

**Comment by nov3mb3r** [comment]
No I have not installed WinAVR or other packages. I retested in a laptop where I have practically nothing, and I have the same problem

**Comment by ceres-c** [comment]
This might be a bit too barbaric, but I'd give a try with [Zadig](https://zadig.akeo.ie/). Try changing the default driver for the Chameleon to libusb, but PLEASE pay attention to not fu*k up any other device

**Comment by nov3mb3r** [comment]
@ceres-c I would appreciate some instructions since I am a total newbie in hardware

**Comment by ceres-c** [comment]
It should be pretty straightforward, you have to select the device you're 100% sure to be the chameleon from the dropdown menu and then pick "libusb" in the selector to the right of the green arrow. Then use the big button "install libusb" (or something like that). The chameleon should then be automatically soft-unplugged from your PC and the device list from device manager will be updated. There you should be able to see something like the output @timokasper has "libusb-win32".
Just be sure to select the right device to force driver update on (the name should be the same as in Device Manager)

**Comment by nov3mb3r** [comment]
@ceres-c Interestingly enough, there are no devices to choose apart from my keyboard and mouse! I am starting to suspect that my chameleon might be faulty

**Comment by ceres-c** [comment]
Check the "List all devices" checkbox in options menu

**Comment by nov3mb3r** [comment]
I already did that before posting :)

**Comment by ceres-c** [comment]
That's quite strange indeed.
It seems a driver issue to me, since an Atmel device is correctly recognized by windows. Could you try with a linux live iso? I can't help you further since USB issues with this kind of devices were one of the reasons I switched from Win to Linux ass a daily driver :)

**Comment by nov3mb3r** [comment]
Will give it a try and make new issues for linux this time if I have problem

**Comment by lvandenb** [comment]
strange , on my machine , using bootloader on Chameleon , I see no com port, and there is Atxmega128A4U
![image](https://user-images.githubusercontent.com/3081470/152346604-e7dabb87-b85a-47a1-8cbe-9c7ceeb5fc67.png)

**Comment by david-oswald** [comment]
@lvandenb that looks like expected behavior: the bootloader is a USB device but does not emulate a COM port. The normal firmware shows then up as COM8 in your case.

**Comment by lvandenb** [comment]
yes , it was only to show a normal/working situation, vs the one on top.
